lib/cdo/rack/global_edition.rb

```diff
@@ -163,17 +162,18 @@ def call
       private def redirectable?(redirect_path)
         return false unless request.get? # only GET request can be redirected
         return false if request.xhr? # only non-AJAX requests should be redirected
+        return false if request.path.include?('/users/auth/')
```

Comment:
Prevents OAuth routes from being redirected to the Global Edition's regional path (`/global/fa`)